osquery/database/plugins/rocksdb.cpp

```diff
@@ -135,7 +135,7 @@ void GlogRocksDBLogger::Logv(const char* format, va_list ap) {
     // logger from trying to make a call back into rocksdb and causing a
     // deadlock
     LoggerForwardingDisabler forwarding_disabler;
-    LOG(INFO) << "RocksDB: " << log_line;
+    VLOG(1) << "RocksDB: " << log_line;
```

Comment:
Please don't change this. :) This is useful information we want to pick up without turning on verbose.